packages/sign_in_with_apple/sign_in_with_apple/pubspec.yaml

```diff
@@ -14,6 +14,9 @@ dependencies:
 
   meta: ^1.3.0
 
+  sign_in_with_apple_platform_interface:
+    path: ../sign_in_with_apple_platform_interface # TODO(tp): This must be changed to the published package
```

Comment:
This we need to fix before merge by publishing the package and using the published one here

Reply:
Done.